tests/testthat/test-driver-databricks.R

```diff
@@ -87,3 +87,20 @@ test_that("supports OAuth M2M in env var", {
   expect_equal(auth$auth_client_id, "abc")
   expect_equal(auth$auth_client_secret, "def")
 })
+
+test_that("dbConnect method handles httpPath aliases (#787)", {
+  local_mocked_bindings(inject = function(...) stop("made it"))
```

Comment:
I would have probably just preferred to mock as `function(...) TRUE` with `expect_true()` but `.valueClassTest()` errors if `dbConnect()` returns a logical.